src/screens/MapScreen.js

```diff
@@ -80,17 +89,47 @@ class MapScreen extends React.Component<Props, State> {
         longitudeDelta: 0.00131,
       },
       isLoading: false,
+      location: null,
+      errorMessage: "",
+      playerPos: {
+        latitude: 0,
+        longitude: 0
+      },
+      playerHeading: 0
     };
   }
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
`componentWillMount` is deprecated. you can keep `this._populateMap()` inside `componentDidMount` but you should find somewhere else to put the Location library calls. would they work if you put them in the constructor of MapScreen?

See https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html for more details.